### ISSUE TITLE
New update 1.7.6.3 issues

### ISSUE THREAD

**Issue by Haircut211** [DillonN/NiceHashMiner#22]
I ran into every issue you can think of. The program loops when executing. My antivirus probably took something out of the new version when installing. I started from scratch many times with my AV disabled. Won't download miners, manually unzipping bins works but then benchmarking crashes the program with every test.

I accidentally overwrote my old version and the only thing I am left with is my config file. 

Literally starting all over for newest version. 

Thank you Devs for your help and knowledgeable miners!

**Comment by Haircut211** [comment]
I attempted from scratch again, issue I'm having is benchmarking everything. It runs the benchmark and then crashes. When I reopen it runs as if the program has never been run before. It doesn't save any of my settings. I noticed it wrote the benchmark info and when progressing to the next benchmark it crashes.

**Comment by Haircut211** [comment]
I attempted v. 1.7.6.0 and that crashes on me too.

**Comment by CreamyG31337** [comment]
weird, maybe try right clicking and run it as administrator?

**Comment by DillonN** [comment]
@Haircut211 please don't use any build before 1.7.6.2 (older versions may get you a temp IP ban!)

Could you post your logs for when you get a crash?

**Comment by Haircut211** [comment]
I got it to run, I manually did all the benchmarks by running them individually as instructed in troubleshooting. Luckily not ip banned or at least I don't think I did. Is the log the .dll file or something else?

**Comment by GongoloMin** [comment]
Running 1.7.6.3, is working very well, in my cfg it is stable.
Win10, 1 RX 570, 1 GTX 1060 6G, 1 GTX 1070, 1 GTX 1080ti.
Really good work, Dillon, congratulations.
Better profit than NHM v.2.0.0.7 (and previous).

**Comment by Haircut211** [comment]
I dug through everything in my folders, I don't see a log. If anyone could help me find that I would appreciate it. I saw someone else post a .zip with the logs folder and some files but I cannot seem to find it

**Comment by Haircut211** [comment]
![untitled](https://user-images.githubusercontent.com/29776562/27709094-7998c9b2-5ce0-11e7-89cc-a644e6cfdb10.jpg)

**Comment by Haircut211** [comment]
also, I was making like $4-5/day with a 1070 and now its been dragging around $3.25. Is there something I am doing wrong to get such low profit?

**Comment by DillonN** [comment]
Hmm it seems NHM is not making a log for some reason. There should be a directory called "logs" right there.

Assuming you have accurate numbers in the benchmarks it should be mining on the most profitable algorithm. I don't have a 1070 so maybe someone who does could chime in on normal numbers, but $3.25 does sound a bit low. Have you tried raising the power limit (e.g. with Afterburner)?

**Comment by Haircut211** [comment]
no, never messed with OC my gpu. I was at $4-5 without it and now its pretty low. I will try it though. Also, I just tried benchmarking my processor and the program gets stuck in a loop. CMD looking window opens and closes in a loop and crashes after a little while and the program opens twice whenever I open it.

**Comment by CreamyG31337** [comment]
I'm getting $5.26 CAD/day so (approx $4.04 USD) dual mining DaggerPascal at the moment on two 1070s (each, $10.55 total)
It really seems like a permissions problem you're having. NHM should not crash during benchmarks but if some files are missing or access denied this could happen. Programs installed to a 'program files' directory need more permissions than if you just created a folder in the root of your drive. Either try running the main program as administrator, or reinstall NHM to a folder in the root of one of your hard drives. Don't just copy it, the strict permissions might copy too. I believe this is a windows 8/10 thing. This is why most programs write .ini and .config files to your appdata folder these days - Windows has got quite strict about programs touching things elsewhere.

**Comment by Haircut211** [comment]
@CreamyG31337 that seems to have worked. It is running normally when I open it and benchmark so far. Fingers crossed, it only did one benchmark but it is looking a lot better than it did earlier.

**Comment by Haircut211** [comment]
And there is now a logs folder in the program. Thank you!

**Comment by CreamyG31337** [comment]
no problem. Profit is bouncing around between $9-11 for my 2 1070s at the moment. Mostly near $11 just took the screenshot at a bad moment. Anyways, I just thought the overclocking settings here might help you. I have 2 Asus founders edition / blower style 1070s. 100% stable like this.
![image](https://user-images.githubusercontent.com/11222932/27711288-f469d8d0-5cdf-11e7-9dd7-267aafc203be.png)

**Comment by ChriscomIT** [comment]
BTW the actual low pay rate for the last one and a half week is normal as it's caused by low prices on the Nicehash market as well as the BTC rate. It's something you or Nicehash isn't able to do anything about it as the market regulates itself. Nice to see the suggestions from Creamy worked out for you regarding the log file. Had some similar issue ones when I installed NHM on a hard disk without the correct system permissions. Starting as admin or installing on the main system drive solved it.

**Comment by Haircut211** [comment]
Sweet thank you everyone for all the help! With afterburner I got profits up to $4.40 USD, temps to 74C and everything is running great! Thank you!

**Comment by Haircut211** [comment]
@CreamyG31337 , I have been playing with afterburner a little bit. Does the core clock being boosted aid with this at all or just the memory clock? Dumb question but I have been tweaking it a little at a time here and there and experimenting.

**Comment by CreamyG31337** [comment]
@Haircut211 core clock is important but it depends on the graphics card and the algo and the difficulty, so just keep experimenting! Generally, either core clock or memory will be a bottleneck, so increasing one or the other will have a greater effect. Dual mining might mean things are more equal, however.
